### PR TITLE
Version 2.0.0b2

### PR DESCRIPTION
**Pull request by dholth**
### Description

Run `rever 2.0.0b2`, update news, recipe.

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: .authors.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   aliases:
   - Mike Sarahan
   - Michael Sarahan
-  num_commits: 98
+  num_commits: 298
```
Comment (dholth) [inline .authors.yml:11]: ??

File: src/conda_package_handling/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "2.0.0a4"
+__version__ = "2.0.0b2"
```
Comment (dholth) [inline src/conda_package_handling/__init__.py:1]: rever appears to have a cache that prevents b1 from being released twice, even after undoing the commit

**Comment by dholth** [comment]
`rever` is not working well for me here... although `rever <version number>` doesn't need further input, its output is surprising.

**Comment by dholth** [comment]
This PR will be out of date, but is an example of the surprising rever behavior.

File: src/conda_package_handling/__init__.py (hunk 2 above)
Comment (dholth) [reply src/conda_package_handling/__init__.py:1]: Did work as expected in a new clone, allowing me to try running rever again for a particular version number.

File: .authors.yml (hunk 1 above)
Comment (kenodegard) [reply .authors.yml:11]: Unfortuantely, this is a "feature" of rever, we can either have a correct `.mailmap` or a correct commit count in `.authors.yml`. We value a correct mailmap more for author deduplication so nothing to see here. The correct commit count is always available via `git shortlog -se`.

File: src/conda_package_handling/__init__.py (hunk 2 above)
Comment (kenodegard) [reply src/conda_package_handling/__init__.py:1]: this is why the `--force` argument is included in the release docs

**Pull request closed without merging**